### ISSUE TITLE
Unnecessary ModifyInstanceAttribute call on ec2 instance creation with disable_api_termination

### ISSUE THREAD

**Issue by ivanpedersen** [terraform-providers/terraform-provider-aws#4939]
### Terraform Version

Terraform v0.11.7
AWS provider v1.24.0

### Community Note
Unnecessary ModifyInstanceAttribute call on ec2 instance launch.

I will provide a fix for this.

### Description

Similar to #4897 Terraform performs an unnecessary ModifyInstanceAttribute call when specifying  disable_api_termination on a new ec2 instance.

### Expected Behavior

Terraform Apply should succeed without errors. 

### Actual Behavior

First Apply fails with:

* aws_instance.app: UnauthorizedOperation: You are not authorized to perform this operation.
	status code: 403

Second Plan gives no diff.

### Terraform Configuration Example

```hcl
provider "aws" {}

resource "aws_instance" "instance" {
  ami                    = "${var.image}"
  subnet_id              = "${var.subnet_id}"
  instance_type          = "t2.small"
  vpc_security_group_ids = ["${var.security_group_ids}"]
  disable_api_termination = true

  tags {
    Name = "disable-api-termination-test"
    Team = "${var.team}"
  }

  root_block_device {
    volume_size = 50
  }

  iam_instance_profile = "${var.iam_instance_profile}"
}
```

### Affected Resource(s)

* aws_instance

**Comment by bflad** [comment]
The fix for this has been merged into master via #4941 and will release with version 1.25.0 of the AWS provider, likely middle next week.

**Comment by bflad** [comment]
This has been released in [version 1.25.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1250-june-27-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
